index.js

```diff
@@ -491,6 +492,9 @@ function fastifyMultipart (fastify, options = {}, done) {
   }
 
   async function cleanRequestFiles () {
+    if (!this.tmpUploads) {
```

Comment:
Particular reason to not set this condition in the `onResponse` hook?

Reply:
I check if this exists before using it.

Reply:
if mean instead of doing this:
```javascript
  fastify.addHook('onResponse', async (request, reply) => {
    if (request.tmpUploads) {
      await request.cleanRequestFiles()
    }
  })
```

Reply:
It's not possible to do so as `cleanRequestFIles` is publicly exposed, and I think it's best to protect it.